src/components/PdfHighlighter.tsx

```diff
@@ -325,15 +317,23 @@ export class PdfHighlighter<T_HT extends IHighlight> extends PureComponent<
   }
 
   renderTables(nextProps?: Props<T_HT>) {
-    const { pdfDocument, renderTables } = nextProps || this.props;
+    const { renderTables, tables } = nextProps || this.props;
+    const pageNumbers = new Set(tables.map(({page_number}) => page_number + 1));
```

Comment:
why do this mapping if we subtract 1 from pageNumber again in each of the lines below